live-examples/js-examples/object/object-create.html

```diff
@@ -1,16 +1,15 @@
 <pre>
-<code id="static-js">function Shape() {
-  this.name = 'Shape 1';
-}
+<code id="static-js">const person = {
+  isHuman: true,
+  sayName: function () {
+    console.log(`My name is ${this.name}`);
+  }
+};
 
-function Rectangle() {
-  Shape.call(this);
-}
+const me = Object.create(person);
+me.name = "Matthew"; // name is now a property set on the me object, but not on the person object
 
-Rectangle.prototype = Object.create(Shape.prototype);
-const rect = new Rectangle();
-
-console.log(rect.name);
-// expected output: "Shape 1"
+me.isHuman == true;
```

Comment:
I don't believe that this line is needed as it is here.

It is not an assignment but a comparison, and the result is not output anywhere. Perhaps you intended for it to be an assignment? Then, the default for `isHuman` is `true` by default so, perhaps it would make more sense to set it to `false`? 

You could then amend the functionality, and name of the `sayName()` function to output something like

```
`My name is ${this.name}. Am I human? ${this.isHuman}`
```

Reply:
I meant to show that the `isHuman` field is accessible on the `me` instance.
I like your idea to show this by using it inside the sayName (I will rename it to `introduceMyself`) method.